objects/grid/_group.scss

```diff
@@ -47,9 +50,24 @@
     margin-left: 0;
     margin-right: 0;
 
-    > [class *= "#{$cactus-prefix}__u"] {
+    > [class *= "o-#{$cactus-prefix}__u"] {
       padding-left: 0;
       padding-right: 0;
     }
   }
+
+  // Reverse
+  // -----------------------------------
+
+  // Usage:
+  // .o-cactus__g.o-cactus__g--reverse
+  //   .cactus__u--1-3...
+
+  &--reverse {
```

Comment:
Not keen on duplication when used in combo i.e. `.o-c__g--small.o-c__g--reverse`.